common.yaml

```diff
@@ -20,19 +20,19 @@
       os_port_master: "{{ infraID }}-master-port"
       os_port_worker: "{{ infraID }}-worker-port"
       # Security groups names
-      os_sg_master: "{{ infraID }}-master"
+      os_sg_master: "{{ infraID }}-controlplane"
```

Comment:
```suggestion
      os_sg_controlplane: "{{ infraID }}-controlplane"
```

Reply:
I'm sure this change would cascade through elsewhere, but if we're changing it in one place might as well remove the term throughout the code base if we can?